### ISSUE TITLE
Encryption and Decryption of Documents Not Working

### ISSUE THREAD

**Issue by MatthewVita** [openemr/openemr#482]
Followed http://open-emr.org/wiki/index.php/Encryption_and_Decryption_of_Documents ... this functionality appears to be broken (unless I am missing something?):

![bug_report_enc_dec_not_working](https://cloud.githubusercontent.com/assets/1715206/22859033/2d0240c4-f09d-11e6-846a-9e720651e8d6.gif)

**Comment by MatthewVita** [comment]
BTW, this is a little bit of a big deal because this feature is "advertised" on the features page:

![image](https://cloud.githubusercontent.com/assets/1715206/22859572/1db4b69a-f0ae-11e6-8204-4a4f0bb3aa86.png)

**Comment by bradymiller** [comment]
Don't use the CCR/CCD/CCDA folders. Those get special treatment. Try doing both the upload/download in a standard folder, such as Medical Record.

**Comment by MatthewVita** [comment]
@bradymiller I've become familiar with CCR/CCD/CCDA, I was just thinking that all encrypted documents were "treated equally".

Anyways, I just tried again in the "Patient Photograph" area and it's not working with my "test_image.png" :(. Can you try it out on your end?

**Comment by bradymiller** [comment]
I tried this on a pdf file and it worked as advertised:
Turned it on in globals
Uploaded pdf doc to document viewer
Went to file in document viewer and downloaded encrypted file
Verified the downloaded file was encrypted
Uploaded encrypted file to document viewer (and toggled it was an encrypted file and entered in the passphrase)
Went to the new file and was able to see it in the document viewer

Note in my case, the pdf file gets embedded in the document viewer at the bottom(ie. does not try to download when click the file in left treee).

Also tried above on a png file and it works. Also, as in the pdf, the image is embedded in the document viewer at the bottom(ie. does not try to download when click the file in left tree).

In your testing, add a .png to the end of the filename test_image before you import and see if that gets it to work.
